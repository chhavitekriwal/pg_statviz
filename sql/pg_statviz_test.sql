CREATE EXTENSION pg_statviz;
SELECT 1 FROM pgstatviz.snapshot();
SELECT count(*)
    FROM pgstatviz.conn t
    JOIN pgstatviz.snapshots s USING (snapshot_tstamp);
