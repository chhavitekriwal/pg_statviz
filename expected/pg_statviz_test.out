CREATE EXTENSION pg_statviz;
SELECT 1 FROM pgstatviz.snapshot();
NOTICE:  created pg_statviz snapshot
 ?column? 
----------
        1
(1 row)

SELECT count(*)
    FROM pgstatviz.conn t
    JOIN pgstatviz.snapshots s USING (snapshot_tstamp);
 count 
-------
     1
(1 row)

